### PR TITLE
angular and the date picker seem to work incredibly well, so lets use…

### PR DESCRIPTION
**Pull request by fredkingham**
… them refs #21

### CODE REVIEW

**Comment by fredkingham** [comment]
screen shot 2019-02-06 at 14 41 04
screen shot 2019-02-06 at 15 27 38

File: rbhl/templates/_helpers/datepicker.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <div
+    {% ifequal style "horizontal" %}
+    class="col-sm-8"
+    {% endifequal %}
+    >
+    {% if user_options %}
+      <div class="input-group">
+    {% endif %}
+        <input type="date" class="form-control"
+               max="2030-12-01"
```
Comment (davidmiller) [inline rbhl/templates/_helpers/datepicker.html]: Do we definitely want to set this to just 11 years in the future?

**Comment by fredkingham** [comment]
Pathway error check is firing if the date is halfway complete

**Comment by fredkingham** [comment]
the date picker field is too large for the clinic log page

**Comment by fredkingham** [comment]
Removed the pathway error check, its an issue with pathways and not the date picker

**Comment by fredkingham** [comment]
fixes the clinic log datepicker size issue

screen shot 2019-02-08 at 19 32 56

**Pull request merged**